airsonic-main/src/main/resources/liquibase/11.0/add-wavpack.xml

```diff
@@ -0,0 +1,15 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="add-wavpack" author="yetangitu">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">select count(*) from transcoding where name='mp3 audio' and (source_formats like 'wv %' or source_formats like '% wv %' or source_formats like '% wv')</sqlCheck>
+        </preConditions>
+        <update tableName="transcoding">
+            <column name="source_formats" valueComputed="(concat((select source_formats from transcoding where name='mp3 audio'), ' wv'))" />
```

Comment:
don't think you need all this.

You can probably just suffice with:
`valueComputed="(concat(source_formats, ' wv'))`

the where clause will take care of the selection

Reply:
True, I just cut and pasted the line from the raw SQL (which I much prefer).